### PR TITLE
Update AboutCommand.ts

### PR DESCRIPTION
**Pull request by killerclaws12**
- Is Bugfix? There was no bug
- Related issues: None
- Closes: None
-

### CODE REVIEW

**Comment by killerclaws12** [comment]
image
will look like this

File: src/commands/settings/AboutCommand.ts [hunk 1]
```diff
@@ -38,33 +38,31 @@ export default class AboutCommand extends BaseCommand {
                 new MessageEmbed()
                 .setAuthor({ iconURL: client.user?.displayAvatarURL(), name: "SudoBot" })
                 .setDescription(`
-                    A free and open source Discord moderation bot, specially created for **The Everything Server**.
-
-                    Copyright (C) ${new Date().getFullYear()} OSN Inc.
-                    This bot comes with ABSOLUTELY NO WARRANTY.
-                    This is free software, and you are welcome to redistribute it under certain conditions.
-                    See the [GNU Affero General Public License v3](https://www.gnu.org/licenses/agpl-3.0.en.html) for more detailed information.
+                    __**A free and open source Discord moderation bot, specially created for The Everything Server**__.
```
Comment (virtual-designer) [inline src/commands/settings/AboutCommand.ts:41]: Maybe add a newline after this line?

**Review by virtual-designer — CHANGES_REQUESTED** [review]
And I think remove the quotes `>` from the fields because for me it is kinda looking weird

File: src/commands/settings/AboutCommand.ts (hunk 1 above)
Comment (killerclaws12) [reply src/commands/settings/AboutCommand.ts:41]: It looks fine to me with the ">".

Comment (virtual-designer) [reply src/commands/settings/AboutCommand.ts:41]: Alright

Comment (virtual-designer) [reply src/commands/settings/AboutCommand.ts:41]: > It looks fine to me with the ">".

But remove those from the fields please.

**Review by virtual-designer — APPROVED** [review]
Nice!

**Pull request merged**